experimenter/experimenter/experiments/migrations/0257_alter_nimbusexperiment_qa_status.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 3.2.23 on 2024-01-16 22:19
+
+from django.db import migrations, models
+
+from experimenter.experiments.constants import NimbusConstants
+
+
+def set_default_qa_status(apps, schema_editor):
+    NimbusExperiment = apps.get_model("experiments", "NimbusExperiment")
+    for experiment in NimbusExperiment.objects.all():
+        if experiment.qa_status is None or experiment.qa_status == "":
+            experiment.qa_status = NimbusConstants.QAStatus.NOT_SET
+            experiment.save()
```

Comment:
I added this directly into the generated migration. Should I have separated this out into a separate manual one?

Reply:
Actually you can just write this as 

```py
NimbusExperiment.objects.filter(qa_status=None).update(qa_status=NimbusConstants.QAStatus.NOT_SET)
```

Reply:
And no that's exactly the right place to put it